client/src/Utils/Contacts/useContactFields.ts

```diff
@@ -45,6 +48,13 @@ const useContactFields = (contactStatus?: InteractedContact['contactStatus']) =>
         : contactType !== ContactType.TIGHT;
 
     const validateContact = (contact: InteractedContact, validationReason: ValidationReason): validValidation | invalidValidation => {
+        const isIdentificationValid = contact.identificationType === IdentificationTypes.ID
+            ? isIdValid(contact.identificationNumber)
+            : isPassportValid(contact.identificationNumber)
+        if(!isIdentificationValid) {
+            return { valid: false, error: `שדה ת.ז. אינו ולידי` };
```

Comment:
change the error massage to be similar to other validation massage 
 ת.ז אינה תקינה
and maybe add and indication for what the wrong id

Reply:
eventually the error is : 
"שגיאה בשורה X : שדה ת.ז. אינו ולידי"
asked ofek what she thinks about it

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1127/commits/1a76f8946e611ba0bdb8b58514d3244bd0de4d65)